cpu/stm32/drivers/vbat/vbat.c

```diff
@@ -0,0 +1,236 @@
+/*
+ * Copyright (C) 2022 Otto-von-Guericke-Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     cpu_stm32
+ * @ingroup     drivers_vbat
+ * @{
+ *
+ * @file
+ * @brief       Implementation of STM32 backup battery monitoring
+ *
+ * @author      Fabian Hüßler <fabian.huessler@ovgu.de>
+ * @}
+ */
+
+#include "board.h"
+#include "periph_conf.h"
+#include "periph/adc.h"
+#include "vbat.h"
+
+/**
+ * @name   Constants depending on CPU line
+ * @{
+ *
+ * @ref VBAT_ADC_SCALE is a scale factor to calculate the right voltage value.
```

Comment:
You find it in the datasheets if you search for "VBAT/" in the document. As far as I understand, this divisor exists because the battery voltage could theoretically be higher than the ADC reference voltage, that means the voltage that the ADC interprets as the maximum sample value.